src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -26,6 +26,9 @@ public abstract partial class PipeStream : Stream
         /// <summary>Characters that can't be used in a pipe's name.</summary>
         private static readonly char[] s_invalidFileNameChars = Path.GetInvalidFileNameChars();
 
+        /// <summary>Characters that can't be used in an absolute path pipe's name.</summary>
+        private static readonly char[] s_invalidPathNameChars = Path.GetInvalidPathChars();
+
         /// <summary>Prefix to prepend to all pipe names.</summary>
         private static readonly string s_pipePrefix = Path.Combine(Path.GetTempPath(), "CoreFxPipe_");
```

Comment:
I thought we were going to use `.NET` as `CoreFxPipe` is not going to be meaningful to many folks. (Eg a machine admin diagnosing some other problem)

Reply:
> I thought we were going to use .NET as CoreFxPipe

This is how we shipped 1.0.  Changing it now would mean you couldn't communicate between the new version and an older version.

Reply:
.NET was the directory that we were adding but in the end we removed. "CoreFxPipe_" is legacy from previous versions and cannot be changed.

Reply:
Ah fine.